packages/form-js-editor/src/features/properties-panel/Util.js

```diff
@@ -56,6 +56,8 @@ export function textToLabel(text) {
   return null;
 }
 
+export const dotPathRegex = /^\w+(\.\w+)*$/;
```

Comment:
I think this is slightly better, but it's up to you

```suggestion
export function isValidDotPath(path) {
  return /^\w+(\.\w+)*$/.test(path);
}
```